src/main/java/dude/task/TaskList.java

```diff
@@ -0,0 +1,112 @@
+package dude.task;
+
+import java.util.ArrayList;
+
```

Comment:
Should there be an empty line here?

Reply:
Thanks for the reviews. 
I think it's needed here according to the coding standard, since java.util.ArrayList and dude.exception.DudeDuplicatedTaskException belongs to different import groups.